Arrowgene.Ddon.GameServer/Handler/PawnGetMypawnDataHandler.cs

```diff
@@ -27,7 +27,7 @@ public override void Handle(GameClient client, StructurePacket<C2SPawnGetMypawnD
             Pawn pawn = client.Character.PawnBySlotNo(req.Structure.SlotNo);
 
 
-            S2CPawn_8_33_16Ntc pcap33 = EntitySerializer.Get<S2CPawn_8_33_16Ntc>().Read(new byte[] {0x0, 0x20, 0xB8, 0xF8, 0x0, 0xDB, 0x3B, 0xCF, 0x0, 0x20, 0xB8, 0xF8, 0x0, 0x5, 0x44, 0x69, 0x61, 0x6E, 0x61, 0x0, 0x6, 0x53, 0x65, 0x65, 0x6C, 0x69, 0x78, 0x0, 0x3, 0x53, 0x3B, 0x52, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xB8, 0xC0, 0xC1});
+            S2CPawnGetPawnProfileNtc pcap33 = EntitySerializer.Get<S2CPawnGetPawnProfileNtc>().Read(new byte[] {0x0, 0x20, 0xB8, 0xF8, 0x0, 0xDB, 0x3B, 0xCF, 0x0, 0x20, 0xB8, 0xF8, 0x0, 0x5, 0x44, 0x69, 0x61, 0x6E, 0x61, 0x0, 0x6, 0x53, 0x65, 0x65, 0x6C, 0x69, 0x78, 0x0, 0x3, 0x53, 0x3B, 0x52, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xB8, 0xC0, 0xC1});
```

Comment:
I actually decoded all these in another change in started to work on on but didn't get around to finishing yet. Not sure if you want to copy those definitions.